client/src/pages/courses.tsx

```diff
@@ -33,18 +47,30 @@ export default function CoursesPage() {
             </Helmet>
             <Grid container spacing={2}>
                 <Grid item xs={isSmallScreen ? 12 : 7}>
-                    <VideoComponent
-                        playlistTitle={courses[0].title}
-                        videoTitle={courses[0].title2}
-                        videoUrl={courses[0].videoUrl}
-                        description={courses[0].description}
-                        videoImage={courses[0].image}
-                    />
+                    {selectedClass?.file ? (
+                        <VideoComponent
+                            playlistTitle={courses[0].title}
```

Comment:
This is because the courses`s team (Matthew and David) already have to modify it, right?

Reply:
Exactly, once the video listing page is done, we will be able to obtain all the data associated with the video, knowing which course a certain user has clicked on.